src/models/backup.ts

```diff
@@ -83,7 +83,9 @@ class Drive {
                     const res = JSON.parse(xhr.responseText);
                     if (res.error) {
                       if (res.error.code === 401) {
-                        localStorage.driveToken = '';
+                        if (navigator.userAgent.indexOf('Chrome') === -1) {
```

Comment:
I still have no idea why we need to check if the current runtime is not Chrome. Can you add a comment in the code here?

Reply:
Does cd95c06 make more sense?